cmd/main.go

```diff
@@ -581,7 +581,7 @@ func setupDB(log logrus.FieldLogger) *gorm.DB {
 	var db *gorm.DB
 	var err error
 	// Tries to open a db connection every 2 seconds for up to 10 seconds.
-	timeout := 10 * time.Second
+	timeout := 30 * time.Second
```

Comment:
Why do we have a timeout at all? Why not just keep trying? A readiness check should be sufficient for communicating that the process is not ready.

Reply:
@rwsu what do you think? You introduced it initially

Reply:
Created https://issues.redhat.com/browse/MGMT-9279 so we can discuss it further

Reply:
It's a small discussion to have, so I think we should just expedite it and remove the timeout unless someone can identify a reason not to.

Reply:
Fair enough, no harm in having a ticket to attach to this PR anyway. Let's wait for input from @rwsu

Reply:
The timeout can be removed. Before, the service would hard fail if the database wasn't ready. So the timeout was added to give more it more time to retry before failing.

Reply:
Ack, done